story/src/2-Icon.stories.module.scss

```diff
@@ -0,0 +1,4 @@
+.icon {
+  width: 24px;
```

Comment:
@Coteh Preferrably divs so that the consumer of the icons has the flexibility to animate it's width & height using css transition or animations easily.